staging/src/k8s.io/client-go/plugin/pkg/client/auth/oidc/oidc.go

```diff
@@ -185,6 +188,28 @@ func (p *oidcAuthProvider) Login() error {
 	return errors.New("not yet implemented")
 }
 
+// UpdateTokenAndCheckConsistent takes a configuration map and updates the token of its cfg, and checks if it is consistent with the given config.
+func (provider *oidcAuthProvider) UpdateTokenAndCheckConsistent(cfg map[string]string) bool {
```

Comment:
Consistent not be a proper name for this method, and it would be better with several unit tests.

Reply:
I was thinking about the function name, can you give me a suggestion?

Unit test is on the way.